sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -1020,6 +1040,41 @@ mod checked {
         builder
     }
 
+    fn setup_bridge_committee_update(
+        mut builder: ProgrammableTransactionBuilder,
+        bridge_shared_version: SequenceNumber,
+    ) -> ProgrammableTransactionBuilder {
+        let bridge = builder
+            .obj(ObjectArg::SharedObject {
+                id: SUI_BRIDGE_OBJECT_ID,
+                initial_shared_version: bridge_shared_version,
+                mutable: true,
+            })
+            .expect("Unable to create Bridge object arg!");
+        let system_state = builder
+            .obj(ObjectArg::SharedObject {
+                id: SUI_SYSTEM_STATE_OBJECT_ID,
+                initial_shared_version: SUI_SYSTEM_STATE_OBJECT_SHARED_VERSION,
+                mutable: true,
+            })
+            .expect("Unable to create System State object arg!");
+
+        // Hardcoding min stake participation to 60.00%
+        // TODO: We need to set a correct value or make this configurable.
+        let min_stake_participation_percentage = builder
+            .input(CallArg::Pure(bcs::to_bytes(&6000u64).unwrap()))
+            .unwrap();
```

Comment:
@longbowlu what stake participation should we set here?

I remember we had discussed this a while ago but can't remember what value we agreed.

Reply:
I don't rmr either. does 7500 look reasonable?